LibTrixi.jl/src/LibTrixi.jl

```diff
@@ -18,6 +18,10 @@ export trixi_is_finished,
 export trixi_step,
        trixi_step_cfptr,
        trixi_step_jl
+export trixi_ndims_cfptr
+export trixi_nelements_cfptr
+export trixi_nvariables_cfptr
+export trixi_get_cell_averages_cfptr
```

Comment:
Is this sufficient? 
Do we actually need the export, or could we just write `LibTrixi.trixi_*`

Reply:
We do not necessarily need the export, you are right. One can always use the qualified version, with `LibTrixi.` prefixed.

However, the `export` also highlights that a given function is part of the public API. That is, it is a user-facing function and not an internal implementation detail. That's why I exported all the functions each time: the `fun`, the `fun_cfptr`, and the `fun_jl`, since they each make sense to be used by a user IMHO.